src/nupic/bindings/engine_internal.i

```diff
@@ -188,6 +192,18 @@
   {
     return nupic::PyArrayRef<nupic::Byte>(self->getOutputData(name)).asNumpyArray();
   }
+
+  // Purge heads of the region's input link buffers
+  void purgeInputLinkBufferHeads()
+  {
+    for (auto & input : self->getInputs())
```

Comment:
That makes sense. Should I add a note about this somewhere, or just make a mental note of it?